crates/cext/src/transpiler/target.rs

```diff
@@ -446,10 +448,11 @@ pub struct TargetEntry {
     operation: StandardOperation,
     params: Option<SmallVec<[Param; 3]>>,
     map: IndexMap<Qargs, Option<InstructionProperties>, ahash::RandomState>,
+    name: Option<String>,
 }
 
 impl TargetEntry {
-    pub fn new(operation: StandardGate) -> Self {
+    pub fn new(operation: StandardGate, name_new: Option<String>) -> Self {
```

Comment:
You can just specify the argument as "name", no need for suffixes here.
```suggestion
    pub fn new(operation: StandardGate, name: Option<String>) -> Self {
```

Reply:
changed.